tests/test_photoCal.py

```diff
@@ -36,12 +35,10 @@
 from lsst.pipe.tasks.photoCal import PhotoCalTask, PhotoCalConfig
 from lsst.pipe.tasks.colorterms import Colorterm, ColortermDict, ColortermLibrary
 from lsst.pipe.base.task_logging import TRACE
+from lsst.meas.algorithms.testUtils import MockLoadReferenceObjects
 
 RefCatDir = os.path.join(getPackageDir("pipe_tasks"), "tests", "data", "sdssrefcat")
```

Comment:
Interesting that we have the same refcats in pipe_tasks and meas_astrom...

Reply:
It is interesting!  Something to consider, I suppose, but not on this ticket...